src/components/navigation/Navbar.scss

```
@@ -0,0 +1,148 @@
+@import '../../constants.scss';
+
+.emptySpacingBox{
```

Prefer space before brace
```suggestion
.emptySpacingBox {
```

---

Done